lib/experimental-default-theme.json

```diff
@@ -161,6 +161,54 @@
 						"size": 42
 					}
 				],
+				"fontStyles": [
```

Comment:
Previously @mtias referred to the intention of not having more presets for now, besides the one for the font family. Would you be able to confirm if that's still the case @mtias?
In the fontStyles and fontWeights I'm not sure if presets bring much value. I guess it may make sense that the themes define for a given font family the weights and styles that are available because depending on the font family one font-weight may be loaded or not, but that may be discussed as part of font loading API.

I guess for font-weight and style we can use normal inline styles without presets e.g: "font-weight: 900;".